cxx/src/CMakeLists.txt

```
@@ -13,7 +13,7 @@ string(CONCAT CMAKE_CXX_FLAGS "-D_GLIBCXX_USE_CXX11_ABI=0 " ${CMAKE_CXX_FLAGS})
 if (CMAKE_BUILD_TYPE MATCHES Debug)
     string(CONCAT CMAKE_CXX_FLAGS "-g -O0 " ${CMAKE_CXX_FLAGS})
     set(INSTALL_SUBDIR "debug")
-else (CMAKE_BUILD_TYPE MATCHES Debug)
+else (CMAKE_BUILD_TYPE MATCHES Release)
```

```suggestion
else ()
```

I think this typo doesn't affect because this line is `else`, not `else if`.